int8 meta7
